frontend/src/utils/utils.js

```diff
@@ -1,3 +1,10 @@
 const categoryOptions = ["All", "Recent", "Celebration", "Thank You", "Inspiration"]
 
-export {categoryOptions}
\ No newline at end of file
+const searchBoards = (boards, searchTerm) => {
```

Comment:
nit: the naming here could be slightly clearer (more action oriented) because you also have some other variables called `searchedBoards` (gets confusing very quickly :) ). 

A better name could be `performSearchOnBoards` or `findBoardsBySearchTerm`